README.md

```diff
@@ -0,0 +1,14 @@
+## Welcome ##
+
+## Pre-Requisite  ##
+1. Java 8
+2. Gradle 5+
+3. Kafka server(https://hub.docker.com/r/spotify/kafka  https://dzone.com/articles/running-apache-kafka-on-windows-os)
+4. Docker
+5. Zipkin
```

Comment:
The prerequisites such as zipkin, kafka & Keycloak are already present in `dependencies` sub folder. Keycloak can be started using docker

Zipkin can be started using `start-zipkin-with-eureka.sh` script